### ISSUE TITLE
[ChoiceGroup]: wcag issues - required and tooltip

### ISSUE THREAD

**Issue by kristiina-richards** [TEDI-Design-System/react#338]
### Description of the problem

1. Required field isn't read as required by JAWS
**Requirement:** [link](https://www.w3.org/WAI/WCAG22/Understanding/info-and-relationships.html)
2. Tooltip cannot be opened by screen readers as space and enter checkes/unchekces the item
**Requirement:** [link](https://www.w3.org/WAI/WCAG22/Understanding/name-role-value.html)
